staging/src/k8s.io/kubectl/pkg/polymorphichelpers/portsforobject.go

```diff
@@ -60,19 +60,31 @@ func portsForObject(object runtime.Object) ([]string, error) {
 }
 
 func getPorts(spec corev1.PodSpec) []string {
-	result := []string{}
+	var result []string
```

Comment:
We are already introducing a new `multiprotocolsforobjects` functionality, why do we need to make changes on `portsforobject`?